src/imp/tracer_imp.js

```diff
@@ -252,7 +252,8 @@ export default class Tracer extends opentracing.Tracer {
         this.addOption('log_field_value_hard_limit', { type: 'int',     defaultValue: 1024 });
 
         // Meta Event reporting options
-        this.addOption('disable_meta_event_reporting', { type: 'bool', defaultValue: true });
+        this.addOption('disable_meta_event_reporting', { type: 'bool', defaultValue: false });
+        this.addOption('meta_event_reporting', { type: 'bool', defaultValue: false });
```

Comment:
I'm not familiar with this pattern of `addOption`, but I wonder if this variable should be an ordinary member (as in `this._meta_event_reporting`) as opposed to an option. I'm not sure what difference this makes, so maybe it's not a great suggestion.

Reply:
I added it here to be consistent with other options and to ensure it's properly initialized.

Reply:
But this is not an option. I'm confused why we'd call it an option. I looked at the definition of `addOption` and it appears not to do much in-so-far as initialization is concerned, other than set a field in the `options` struct. Why not use the tracer itself to store this? Apologies if I'm showing off a great misunderstanding of JS right now.

Reply:
Mostly because we might want to use it as an option in tests or internally. We could move it somewhere else to try and hide it, but it's not like there's any real difference other than, again, being consistent with other options.